templates/README.md

```diff
@@ -0,0 +1,29 @@
+# Snowflake Native SDK for Connectors Template
+
+Snowflake Native SDK for Connectors Template is a simple connector native application using Snowflake Native SDK for Connectors.
+The purpose of this template is to provide a simple connector that can be used as a starting point when developing a new connector.
+It can be deployed to a Snowflake environment out of the box and then each of its features can be customized according to more specific requirements.
+It is recommended to get familiar with the [Snowflake Native SDK for Connectors tutorial](link to tutorial) 
```

Comment:
Maybe let's add `TODO:` in places where we need to add something in the future linked to the `(link to tutorial)`. It will be easier to find this place.

Reply:
no need for this imo. This file is small enough to fit on one screen and intellij shows warning: cannot resolve on those anyway so they are yellow. but ill add them just in case

Reply:
I added them in other files to change them before the final merge, so maybe let's be consistent, thx!